### PR TITLE
Make task Date and Time

### PR DESCRIPTION
**Pull request by alex39292**
Date and Time task

### CODE REVIEW

File: date_and_time/task1.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+const date = new Date();
+console.log(date.getDate());
```
Comment (SiarheiRylach) [inline date_and_time/task1.js]: current date, not day

File: date_and_time/task2.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+const date = new Date();
+console.log('Дата: ' + date.getDate() + '.' + date.getMonth() + '.' + date.getFullYear() + 'г.')
```
Comment (SiarheiRylach) [inline date_and_time/task2.js]: date.getDate() + 1, because months is counted from 0

File: date_and_time/task2.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+const date = new Date();
```
Comment (SiarheiRylach) [inline date_and_time/task2.js]: Also I don't see a task from https://learn.javascript.ru/datetime

Comment (alex39292) [reply date_and_time/task2.js]: Added this task

File: date_and_time/task2.js (hunk 2 above)
Comment (alex39292) [reply date_and_time/task2.js]: fixed! Forgot about it)

File: date_and_time/task1.js (hunk 1 above)
Comment (alex39292) [reply date_and_time/task1.js]: fixed!

File: date_and_time/task2.js (hunk 2 above)
Comment (SiarheiRylach) [reply date_and_time/task2.js]: well, I made a mistake. You should make `date.getMonth()` + 1, not `date.getDate()`.

File: date_and_time/formatDate.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+const date = new Date();
+
+function formatDate(date) {
```
Comment (SiarheiRylach) [inline date_and_time/formatDate.js:3]: tests for your function don't pass.
https://plnkr.co/edit/gun7gTffKiQvAB6H?p=preview&preview

File: date_and_time/task2.js (hunk 2 above)
Comment (alex39292) [reply date_and_time/task2.js]: Yeah, I understand. Fixed it

File: date_and_time/formatDate.js (hunk 4 above)
Comment (alex39292) [reply date_and_time/formatDate.js:3]: Fixed it. It was problem with output year veriable. It returnded full year (2014), but not (14).

**Pull request merged**